src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/CaseDataExtractor.java

```diff
@@ -61,6 +61,7 @@ private static ScannedDocument mapDocument(Document document) {
             document.subtype,
             ZonedDateTime.ofInstant(document.scannedAt, ZoneId.systemDefault()).toLocalDateTime(),
             new CcdDocument(String.valueOf(document.url)),
+            ZonedDateTime.ofInstant(document.deliveryDate, ZoneId.systemDefault()).toLocalDateTime(),
```

Comment:
not for this PR as this might be used throughout entire project, but just noting - we should perhaps ensure it is `Europe/London` as discovered couple times now.

Reply:
👍 
I think it's better to do it as part of another PR.